### DISCUSSION TITLE
What is the format `tracing-flame`'s textual representation?

### DISCUSSION THREAD

**Discussion by byte-sourcerer** [tokio-rs/tracing#3086] [Q&A]
I would like to generate the flaming graph by using `tracing-flame-0.2.0`, but the result is weird. I would like to know the text format of it, so I can figure out whether it's a bug of `tracing-flame`🤔.

The text is like:

```
ThreadId(18) 216575291
ThreadId(18);A::B::C::put_batch 16042
ThreadId(18);A::B::C::put_batch;A::D::transaction::write_batch 294917
ThreadId(18);A::B::C::put_batch 4625
ThreadId(18);A::B::C::put_batch;A::D::transaction::commit 1875
ThreadId(18);A::B::C::put_batch 2083
ThreadId(18) 17163333
ThreadId(18);A::B::C::put_batch 959
ThreadId(18);A::B::C::put_batch;A::D::transaction::commit 12291
ThreadId(18);A::B::C::put_batch;A::D::transaction::commit;A::state::states::write_batch 7667
ThreadId(18);A::B::C::put_batch;A::D::transaction::commit 708
ThreadId(18);A::B::C::put_batch 459
...
```

the flamegraph I generated is:

![CleanShot 2024-09-23 at 22 57 02@2x](https://github.com/user-attachments/assets/134d4c71-a0a8-44e0-9230-606aef66766f)

**Comment by kaffarell** [comment]
How did you generate the flamegraph? Using `inferno` (https://github.com/tokio-rs/tracing/tree/master/tracing-flame#generating-the-image)?
